core/src/main/scala/tech/beshu/ror/accesscontrol/logging/AccessControlLoggingDecorator.scala

```diff
@@ -115,7 +118,7 @@ class AccessControlLoggingDecorator(val underlying: AccessControl, auditingTool:
 
 object AccessControlLoggingDecorator {
 
-  private implicit val responseContextShow: Show[ResponseContext] = {
+  private implicit def responseContextShow(implicit headerShow:Show[Header]): Show[ResponseContext] = {
```

Comment:
why not to pass here whole LoggingContext? In future we'd like to add more data to LoggingContext, so there will be no need to change signature of this method.